broadinstitute/ddp-study-server
# DDP-8599: exporting option details in question defs

This adds option details text and whether a question has option details to the ElasticSearch export.  I've written an automated test for this, but I'm not sure how to test this on a running system.  Also, what would the process need to be to have all the question definitions currently in production get re-exported?

TO TEST: 
1. Trigger export of study in dev
2.  confirm updated definitions were written for option details
3. ,confirm updated text is used by the DSM study export feature
